File: sigma/sigma_case_management/page/case_dashboard/case_dashboard.py
import frappe

@frappe.whitelist()
def get_case_summary():
    return {
        "open": frappe.db.count("Case", {"status": "Open"}),
        "resolved": frappe.db.count("Case", {"status": "Resolved"}),
        "awaiting_legal": frappe.db.count("Case", {"status": "Awaiting Legal"})
    }

File: sigma/sigma_case_management/page/case_dashboard/case_dashboard.js
frappe.pages['case-dashboard'].on_page_load = function(wrapper) {
    const page = frappe.ui.make_app_page({
        parent: wrapper,
        title: 'Case Management Dashboard',
        single_column: true
    });

    $(frappe.render_template("case_dashboard", {})).appendTo(page.body);

    frappe.call({
        method: "sigma.sigma.sigma_case_management.page.case_dashboard.case_dashboard.get_case_summary",
        callback: function(r) {
            if (r.message) {
                $("#case-stats").html(`
                    <div><b>Open Cases:</b> ${r.message.open}</div>
                    <div><b>Resolved Cases:</b> ${r.message.resolved}</div>
                    <div><b>Pending Legal:</b> ${r.message.awaiting_legal}</div>
                `);
            }
        }
    });
};
